index.html

```diff
@@ -459,146 +459,53 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
   </section>
 
   <!-- Profiling mechanism -->
-
   <section id="profiling-mechanism">
-    <h2 id="the-profiling-profile">Profiling Mechanism</h2>
+    <h2>Profiling Mechanism</h2>
     <p>
-      This section describes a generic mechanism to define a
-      profile of the <a>WoT Thing Description</a> in a unambiguous way.
+      In order to denote that a given
+      <a href="https://www.w3.org/TR/wot-architecture/#dfn-thing">Web Thing</a>
+      conforms to a particular profile, its Thing Description
+      [[wot-thing-description]] MUST include a <code>profile</code> member whose
+      value is set to a valid profile identifier corresponding to that profile.
     </p>
     <p>
-      The W3C <a>WoT Thing Description</a> specification defines a
-      formal language, i.e. a set of vocabulary terms (keywords),
-      a set of classes that are built from these keywords, and a
-      set of additional rules, that define constraints on
-      permitted values and keyword presence (mandatory / optional)
-      dependent on the context where the keyword is used. In
-      addition the <a>WoT Thing Description</a> specification defines relationships and
-      corresponding cardinalities between these classes.
+      A profile identifier MUST be a valid URI [[RFC3986]].
     </p>
+    <pre class="example">
+    {
+      "@context": "https://www.w3.org/2019/wot/td/v1",
+      "id": "urn:dev:ops:32473-WoTLamp-1234",
+      "profile": "https://www.w3.org/2021/wot/profile/core",
+      "title": "My Lamp",
+      "description": "A web connected lamp",
+      ...
+    }
+    </pre>
     <p>
-      The <a>WoT Thing Description</a> specification already has some constraints,
-      but there is a wide variety of variations that are left to
-      the interpretation or the discretion of an implementer. The
-      rationale for the <a>Core Profile</a> is not to forbid
-      complex things, rather to enable statements like:
+      In order to denote that a given
+      <a href="https://www.w3.org/TR/wot-architecture/#dfn-thing">Web Thing</a>
+      conforms to multiple profiles, its Thing Description
+      [[wot-thing-description]] MUST include a <code>profile</code> member whose
+      value is set to an array of valid profile identifiers corresponding to the
```

Comment:
I would suggest a `string` only for a single profile. An `array` if multiple profiles are implemented. We have multiple places in the TD for such kind of approach (e.g., `security`, `@type`, ...)

Reply:
This was what I intended to specify, so I need to revisit the wording to make this more clear.

Reply:
Done.